.github/workflows/securityAlertsReview.yml

```diff
@@ -0,0 +1,194 @@
+name: Security Alerts Review
+
+on:
+  push:
+    paths:
+      - 'src/**.sol'
+      - ".github/workflows/securityAlertsReview.yml"
+  pull_request:
+    types:
+      - synchronize
+      - reopened
+      - ready_for_review
+  pull_request_review:
+    types:
+      - submitted
+  workflow_dispatch:
+
+jobs:
+  check-security-alerts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Fetch PR Number
+        id: fetch_pr
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PR_NUMBER=$(curl -s -H "Authorization: token ${GITHUB_TOKEN}" \
+            "https://api.github.com/repos/${{ github.repository }}/pulls?state=open" | jq -r '.[0].number')
+
+          if [[ -z "$PR_NUMBER" || "$PR_NUMBER" == "null" ]]; then
+            echo "No open PR found, skipping check."
+            exit 0
+          fi
+
+          echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_ENV
+          echo "PR number: $PR_NUMBER"
+
+      - name: Fetch Security Alerts for PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "Fetching security alerts for PR #${PR_NUMBER}..."
+          
+          # Fetch security alerts via GitHub API
+          ALERTS=$(curl -s -H "Authorization: token ${GITHUB_TOKEN}" \
+            "https://api.github.com/repos/${{ github.repository }}/code-scanning/alerts?pr=${PR_NUMBER}")
+
+          # Log raw API response for debugging
+          echo "Raw API Response:"
+          echo "$ALERTS"
+
+          # Extract unresolved alerts (open alerts)
+          UNRESOLVED_ALERTS=$(echo "$ALERTS" | jq -c '[.[] | select(.state == "open") ]' || echo "[]")
+          # Extract dismissed alerts without comments (empty dismissed_comment)
+          DISMISSED_ALERTS=$(echo "$ALERTS" | jq -c '[.[] | select(.state == "dismissed" and (.dismissed_comment == null or .dismissed_comment == ""))]' || echo "[]")
+          # Extract dismissed alerts with comments (successful dismissals)
+          COMMENTED_ALERTS=$(echo "$ALERTS" | jq -c '[.[] | select(.state == "dismissed" and (.dismissed_comment != null and .dismissed_comment != ""))]' || echo "[]")
```

Comment:
maybe rename to "RESOLVED_ALERTS" ?

Reply:
changed it to "DISMISSED_AND_COMMENTED_ALERTS". IMO "resolved" should only refer to alerts that weren't dismissed and instead they were reviewed and fixed by a developer.

Reply:
updated